src/sim_engine.rs

```diff
@@ -6,15 +6,21 @@ use blockdev::BlockDevs;
 
 use engine::Engine;
 
-use pool::StratisPool;
-
+use std::fmt::Debug;
 use std::path::Path;
 use std::collections::BTreeMap;
 use std::iter::FromIterator;
 
 use types::StratisResult;
 
 
+pub trait StratisPool: Debug {
```

Comment:
It also might be good (now that struct Pool is gone) to just call this trait Pool. i.e make the generic traits the short, generic words w/o prefix.

This will happen to let StratisEngine call its struct that implements the Pool trait be called StratisPool :smile: